src/TensorFlowNET.Core/Keras/Layers/ILayersApi.cs

```diff
@@ -48,7 +48,7 @@ public partial interface ILayersApi
             string data_format = "channels_last",
             int dilation_rate = 1,
             int groups = 1,
-            string activation = null,
```

Comment:
I think we should keep the function with `string` as the type of activation, because using string to specify an activation is quite common in tensorflow. Though the implicit conversion from `string` to `ActivationAdapter` has been provided, it may cause some confusion to users. Maybe splitting them to two functions is better, the one with `string` and the other with `ActivationAdapter`.